zerver/views/home.py

```diff
@@ -254,6 +255,23 @@ def home_real(request: HttpRequest) -> HttpResponse:
                                'embedded': narrow_stream is not None,
                                },)
     patch_cache_control(response, no_cache=True, no_store=True, must_revalidate=True)
+
+    uri_host_source = urlparse(user_profile.realm.uri).netloc
+    ws_source = ''
+    if uri_host_source:
+        ws_source = ('ws://%s/' % (uri_host_source))
+    csp_policy = ("object-src 'none'; "
+                  "default-src 'none'; "
+                  "img-src https://* 'self'; "
+                  "font-src 'self'; "
+                  "frame-src  https://*; "
+                  "media-src 'self'; "
+                  "connect-src http://localhost:9994/ ws://localhost:9994/ %s 'self'; "
```

Comment:
I think the `connect-src` part we should only include if `settings.DEVELOPMENT`.

Reply:
its needed in prod as well. The %s part is used in prod for websockets.
I thing what we can do is add a better conditional statement?